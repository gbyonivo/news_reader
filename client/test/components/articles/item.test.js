import React from 'react';
import renderer from 'react-test-renderer';
import Item from '../../../src/components/articles/item';

describe('Item component', () => {
  it('renders correctly', () => {
    const item = {
      title: 'a',
      link: 'b',
      images: []
    }
    const actual = renderer.create(
      <Item item={item}/>
    );
    expect(actual).toMatchSnapshot();
  });
  it('renders correctly with images', () => {
    const item = {
      title: 'a',
      link: 'b',
      images: ['a', 'b']
    }
    const actual = renderer.create(
      <Item item={item}/>
    );
    expect(actual).toMatchSnapshot();
  });
})