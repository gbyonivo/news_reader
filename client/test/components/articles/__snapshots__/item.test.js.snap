// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item component renders correctly 1`] = `
<div
  className="item"
>
  <h2>
    <a
      href="b"
    >
      a
    </a>
  </h2>
</div>
`;

exports[`Item component renders correctly with images 1`] = `
<div
  className="item"
>
  <h2>
    <a
      href="b"
    >
      a
    </a>
  </h2>
  <img
    src="a"
  />
  <img
    src="b"
  />
</div>
`;
